README.md

```diff
@@ -10,7 +10,7 @@ K8Dash is the easiest way to manage your Kubernetes cluster. Why?
 
 
 ## Click the video below to see K8Dash in action
-[![K8Dash - Kubernetes Dashboard](https://raw.githubusercontent.com/herbrandson/k8dash/master/docs/videoThumbnail.png)](http://www.youtube.com/watch?v=u-1jGAhAHAM "K8Dash - Kubernetes Dashboard")
+[![K8Dash - Kubernetes Dashboard](https://github.com/indeedeng/k8dash/blob/master/docs/videoThumbnail.png)](http://www.youtube.com/watch?v=u-1jGAhAHAM "K8Dash - Kubernetes Dashboard")
```

Comment:
I believe this needs to be `https://raw.githubusercontent.com/indeedend/k8dash/master/docs/videoThumbnail.png`
The link `https://github.com/indeedeng/k8dash/blob/master/docs/videoThumbnail.png` goes to an actual GitHub page, rather than the raw file.

Same goes for the other links below

Reply:
The raw link resolves to a 404 and at the moment I am not sure why.